src/me/drton/jmavsim/LogPlayerSensors.java

```diff
@@ -87,6 +87,21 @@ public boolean isGPSUpdated() {
     }
 
     @Override
+    public double getSonarDist() {
+        return 0; // TODO: implement me
+    }
+
+    @Override
+    public FlowData getFlowData() {
+        return null; // TODO: implement me
+    }
+
+    @Override
+    public boolean isFlowUpdated() {
+        return false; // TODO: implement me
+    }
```

Comment:
PX4LogReader expects px4log files which were replaced by ulg files, so I do not think this is worth implementing.